bluemix/terminal/table.go

```diff
@@ -108,10 +114,10 @@ func (t *PrintableTable) printRow(row []string) {
 
 func (t *PrintableTable) cellValue(col int, value string) string {
 	padding := ""
-	if col < len(t.headers)-1 {
```

Comment:
maxSizes is more fluid than the headers, I don't think we want to change this.

Reply:
This is necessary for headerless tables where the only reliable count of the number of columns is the length of maxSizes. Would a rename of the var to something like `maxColumnLengths` make you more comfortable?

Reply:
Agreed. `maxSizes` can grow based on the number of rows added and not col which may give different results as the table grows. Lets revert this change.

Reply:
What should be done about [this use case](https://github.com/IBM-Cloud/ibm-cloud-cli-sdk/blob/7d34e511990ecbc0c62c08ee55305d3841e2b552/bluemix/terminal/table_test.go#L52-L59)? It appears to be a supported use case as there is a test to fix the behaviour, but reverting this line as you suggest will remove all padding from between the columns, which is clearly not okay.

Reply:
Yeah I see your point here. I think the change is fine here since the padding should be dependent on the number of rows added. Headers are optional which shouldn't be coupled effect if padding should be added. Thoughts @steveclay

Reply:
I did not recall a headerless scenario. That is not something that would pass CLI standards for the user interface, but is apparently allowed in the code. I think this change is fine, therefore.